scripts/resurrect-repositories.rb

```diff
@@ -55,7 +55,13 @@ def find_git_remote_url(git_cmd, remote_name)
 end
 
 def find_git_repos_from_disk(path)
-  Dir.glob("#{path}/**/.git").map { |d| d.sub('/.git', '') }.compact.sort.uniq
+  stderr = "/tmp/#{File.basename(__FILE__)}-stderr"
+  paths = `find '#{path}' -name .git -type d -exec dirname {} \\; 2>#{stderr}`
+  File.read(stderr).split("\n").each_with_index do |line, i|
```

Comment:
Instead of opening the file, splitting line-by-line, can't we just `cat` the file using the system shell command?

Reply:
Done